cmdutil/svclog/logger.go

```diff
@@ -83,3 +84,19 @@ func (sl saramaLogger) Printf(format string, args ...interface{}) {
 	format = strings.TrimSpace(format)
 	sl.FieldLogger.Printf(format, args...)
 }
+
+// NewNullLogger returns a logger that discards the output useful for testing
```

Comment:
Since you mentioned "for testing" here, there is actually a package already with a null logger for testing: https://godoc.org/github.com/heroku/x/testing/testlog. It also includes useful assertions for testing structured log output.